scripts/create-s3-bucket.bash

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Deploys in local host
+#
+#
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit
+set -o nounset
+set -o pipefail
+IFS=$'\n\t'
+
+docker run \
+-v /etc/ssl/certs:/etc/ssl/certs:ro \
+--network host \
+-e MC_HOST_local="https://${S3_ACCESS_KEY}:${S3_SECRET_KEY}@${S3_ENDPOINT}" \
```

Comment:
MINOR: as a general rule, we agreed that when written in a script, we should use long CLI arguments, e.g. here you should use ``--environment`` instead of ``-e``